platform_versioned_docs/version-23.4/getting-started/rnaseq.mdx

```diff
@@ -0,0 +1,742 @@
+
+  ![General run information](./quickstart-demo/assets/general-run-details.gif)
+
+  The **General** panel displays top-level information about a pipeline run:
+
+  - Unique workflow run ID
+  - Workflow run name
+  - Timestamp of pipeline start
+  - Pipeline version and Git commit ID
+  - Nextflow session ID
+  - Username of the launcher
+  - Work directory path
+
+  #### View details for a task
+
+  Scroll down the page to view:
+
+  - The progress of individual pipeline **Processes**
+  - **Aggregated stats** for the run (total walltime, CPU hours)
+  - **Workflow metrics** (CPU efficiency, memory efficiency)
+  - A **Task details** table for every task in the workflow
+
+  The task details table provides further information on every step in the pipeline, including task statuses and metrics.
+
+  #### Task details
+
+  Select a task in the task table to open the **Task details** dialog. The dialog has three tabs: **About**, **Execution log**, and **Data Explorer**.
+
+  The **About** tab includes:
+
+  1. **Name**: Process name and tag
+  2. **Command**: Task script, defined in the pipeline process
+  3. **Status**: Exit code, task status, and number of attempts
+  4. **Work directory**: Directory where the task was executed
+  5. **Environment**: Environment variables that were supplied to the task
+  6. **Execution time**: Metrics for task submission, start, and completion time
+  7. **Resources requested**: Metrics for the resources requested by the task
+  8. **Resources used**: Metrics for the resources used by the task
+
+  ![Task details window](./quickstart-demo/assets/task-details.gif)
+
+  #### Execution log
+
+  The **Execution log** tab provides a real-time log of the selected task's execution. Task execution and other logs (such as stdout and stderr) are available for download from here, if still available in your compute environment.
+
+  #### Task work directory in Data Explorer
+
+  If a task fails, a good place to begin troubleshooting is the task's work directory.
+
+  Nextflow hash-addresses each task of the pipeline and creates unique directories based on these hashes. Instead of navigating through a bucket on the cloud console or filesystem to find the contents of this directory, use the **Data Explorer** tab in the Task window to view the work directory.
+
+  Data Explorer allows you to view the log files and output files generated for each task in its working directory, directly within Platform. You can view, download, and retrieve the link for these intermediate files in cloud storage from the **Data Explorer** tab to simplify troubleshooting.
+
+  ![Task data explorer](./quickstart-demo/assets/sp-cloud-task-data-explorer.gif)
+
+</details>
+
+## Tertiary analysis with Data Studios
+
+Set up an environment for tertiary analysis of your pipeline data and enable collaboration with other workspace participants. 
```

Comment:
```suggestion
Set up an environment for tertiary analysis of your pipeline data and enable collaboration with other workspace participants. 
```

This reads like an instruction to do that right now- maybe it should be combined with the section title?